test/e2e/node_drain.go

```diff
@@ -641,3 +699,26 @@ func getDrainAndDeleteInterval(deleteInterval []interface{}, drainTimeout *metav
 	res := []interface{}{intervalDuration.String(), deleteInterval[1]}
 	return res
 }
+
+func unblockNodeVolumeDetachment(ctx context.Context, bootstrapClusterProxy framework.ClusterProxy, cluster *clusterv1.Cluster, waitControlPlaneIntervals, waitWorkerNodeIntervals []interface{}) {
```

Comment:
q: should this be in test/e2e/node_drain_test.go, given that this is pluggable and somehow related to how additional resources are created?

Reply:
We default to this function, so it has to be in this file. This way we allow to overwrite it.